packages/services/src/workers/notification-sender.js

```diff
@@ -0,0 +1,35 @@
+import emailClient from '@aragonone/court-backend-shared/helpers/email-client'
+import { UserNotification } from '@aragonone/court-backend-server/build/models/objection'
+import * as notificationScanners from '../models/notification-scanners'
+
+/**
+ * This worker loops over all unprocessed notification DB entries
+ * and sends an associated email
+ */
+export default async function (logger) {
+  const notifications = await UserNotification.query().whereNull('sentAt')
+  for (const notification of notifications) {
+    await trySendNotification(logger, notification)
+  }
+}
+
+export async function trySendNotification(logger, notification) {
+  if (notification.sentAt != null) return
+  notification = await notification.$fetchGraph('[user.email, type]')
+  const { user, type: { model } } = notification
+  const scanner = notificationScanners[model]
+  if (!scanner) throw `Notification scanner ${model} not found.`
```

Comment:
Instead of throwing we could use `logger` to log an error here cause otherwise, it will stop the execution of all the notification for one wrong DB entry

Reply:
Good idea, switched to `logger.error` + `return`
Actually I was thinking about services in general, I think it would make more sense to run each service as a separate container. Then I will have more visibility on kubernetes when individual processes are failing and also kubernetes will always try to restart the container if the main process fails. Currently if any worker fails we will not hear about it.